mule-packager/src/main/java/org/mule/tools/api/util/exclude/GlobMatcher.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.api.util.exclude;
+
+import java.nio.file.Path;
+import java.nio.file.PathMatcher;
+
+import static java.nio.file.FileSystems.getDefault;
+
+class GlobMatcher implements PathMatcher {
```

Comment:
public class

Reply:
Also could you please put a name a little bit more descriptive and/or add doc

Reply:
There is no reason for this class to be other access control than package private, since no other classes use it

Added the doc